src/main/java/org/broadinstitute/hellbender/utils/GATKProtectedMathUtils.java

```diff
@@ -47,6 +48,16 @@ public static double logSumExp(final Collection<Double> values) {
         return max + Math.log(sum);
     }
 
+    /**
+     * Returns log(max(abs(x), minLogArg))
+     * @param x argument of log (any double value)
+     * @param minLogArg minimum argument to pass to log (must be positive but not checked)
+     * @return any double value.
+     */
+    public static double safeLog(final double x, final double minLogArg) {
```

Comment:
Need to talk me thru this method and is usages, I don't understand what you try to achieve with it.